vision.md

```diff
@@ -0,0 +1,52 @@
+Stated simply...
+
+**NUnit 3 is a world-class platform for individuals and teams using a test-based approach to .NET software development.**
+
```

Comment:
I think this should be our one missions statement, an elevator pitch for NUnit that sums up everything below. I think the current statement is too generic and doesn't actually say anything. I am going to propose some words to get it started, but feel free to rework and remix,

> NUnit aims to be a best in breed test framework for .NET that enables end to end testing of .NET projects by being unopinionated and allowing many testing styles.

I like the `unopinionated` because other test frameworks are often regarded as opinionated, ie forcing you to write tests one way and I think we should embrace being open. That is why I use NUnit and joined the team in the first place 😄 

We might also want to add something about supporting all commonly used .NET platforms and maintaining support for older platforms.